File: classes/parte-7-paradigmas/110-orientado-a-objetos-clases-objetos-y-estado/implementaciones/python/main.py
import sys


class Contador:
    def __init__(self):
        self.cuenta = 0

    def incrementar(self):
        self.cuenta += 1


n = int(sys.stdin.readline())
c = Contador()
for _ in range(n):
    c.incrementar()
print(f"cuenta={c.cuenta}")

File: classes/parte-7-paradigmas/110-orientado-a-objetos-clases-objetos-y-estado/implementaciones/java/Main.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    static class Contador {
        int cuenta = 0;
        void incrementar() { cuenta++; }
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine().trim());
        Contador c = new Contador();
        for (int i = 0; i < n; i++) c.incrementar();
        System.out.println("cuenta=" + c.cuenta);
    }
}
